### ISSUE TITLE
Account Issue

### ISSUE THREAD

**Issue by Katrinellen** [RenderHeads/UnityPlugin-AVProVideo#1341]
Problem description:

I've bought the plug-in, but in Unity I see the watermark on the video. 
I couldn't find any place to setup the license. 
Where should I go?

Device (which devices are you having the issue with - model, OS version number):

Media (tell us about your videos - number of videos, resolution, codec, frame-rate, example URLs):

System Information:

AVPro Video: v2.6.5 (plugin v2.6.3f1-trial)
Target Platform: Android
Unity: v2021.3.11f1 WindowsEditor
OS: Desktop - 82JH (LENOVO) - Windows 11  (10.0.22000) 64bit - English
CPU: 11th Gen Intel(R) Core(TM) i7-11800H @ 2.30GHz - 16 threads - 16151KB
GPU: NVIDIA GeForce RTX 3060 Laptop GPU - NVIDIA - Direct3D 11.0 [level 11.1] - 6023KB - 16384

**Comment by Chris-RH** [comment]
Hi @Katrinellen 

Did you follow the upgrade steps?
https://www.renderheads.com/content/docs/AVProVideo/articles/install.html#upgrade-steps

Which version did you purchase?
You can check which version you have installed by adding an MediaPlayer component to your scene and clicking on the “About / Help” button in the Inspector for that component. The version number is displayed in this box.

The Core and Ultra editions of AVPro Video have no watermarks for any platforms. If you use one of the platform specific Core editions (eg AVPro Video - Core Windows Edition) then you will not see the watermark on the platform you purchased for, but you will see the watermark on the other platforms. For example if you purchased AVPro Video - Core Android Edition then you will still see the watermark in the Unity editor as this is running on Windows/macOS, but the videos played back when you deploy to your Android device will be watermark-free.

**Comment by Katrinellen** [comment]
Hi!
Sorry, but I still have some problems with the plug-in. 
![Screen_Unity](https://user-images.githubusercontent.com/116175767/199750056-bf19be1a-2dce-47e3-9913-2400a253e5ae.png)
![Screen_Unity_2](https://user-images.githubusercontent.com/116175767/199750061-f4043861-fcbd-4389-862b-c57c5752340d.png)

As you can see, I have theese screen in Unity's Inspector. It's seems I still have the "Trial version" and I don't understand why.
When I press play in Unity the streaming works (with the watermark, but its ok) but when I build the APK in my Meta Quest 2 it doesn't work at all. 

Can you help me again? 

Thanks a lot

**Comment by Chris-RH** [comment]
Which version did you purchase?
Did you follow the upgrade steps above?

**Comment by Katrinellen** [comment]
I've purchased "AVPro Video - Core Android Edition" and the version is this one: 
![Screen_Unity_3](https://user-images.githubusercontent.com/116175767/199767191-6949a7b0-df47-443c-857d-2771897eaf05.png)
and I've followed all the steps  above

**Comment by Ste-RH** [comment]
> but when I build the APK in my Meta Quest 2 it doesn't work at all

Can you send over a full logcat output please?

**Comment by Chris-RH** [comment]
The Core Android version will say trial in the editor as you have seen above, but it will not have a watermark when you use it on an android device.

**Comment by Katrinellen** [comment]
I've tried with a new fresh instance but it still says "trial". Also in the screenshots above there's the version of the plugin.

My question is not about the watermark, I'm just asking if it's normal that in Unity the plugin is set as "trial" in the version number.

**Comment by Chris-RH** [comment]
Yes, because the Windows/editor version is still considered trial, so have trial in the version number and will also have the watermark.  When you put it on the android device, it will not have a watermark.

**Comment by Katrinellen** [comment]
Ok, thanks
maybe I'm doing something wrong, because on Android it still doesn't work. 
I'll try an other way.

**Comment by Ste-RH** [comment]
Do you mean that video playback is not working on Android?  Or is does, but with a watermark?

**Comment by Katrinellen** [comment]
It seems the problem is the URL, the plug in works with HTTPS only. 
Now everthing seems ok. 
Thanks for the support!

**Comment by Ste-RH** [comment]
Great to hear that you have tracked it down.
